pkg/controller/plan/kubevirt.go

```diff
@@ -1132,12 +1133,20 @@ func (r *KubeVirt) virtualMachine(vm *plan.VMStatus) (object *cnv.VirtualMachine
 	}
 
 	var ok bool
-	object, ok = r.vmTemplate(vm)
-	if !ok {
-		r.Log.Info("Building VirtualMachine without template.",
+	object, err = r.vmPreference(vm)
+	if err != nil {
+		r.Log.Info("Building VirtualMachine without a preference.",
```

Comment:
it is a bit ambiguous this way, how about replacing 'a preference' with VirtualMachinePreference?